x/trigger/client/cli/tx.go

```diff
@@ -0,0 +1,261 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/provenance-io/provenance/x/trigger/types"
+)
+
+// NewTxCmd is the top-level command for trigger CLI transactions.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Aliases:                    []string{"t"},
+		Short:                      "Transaction commands for the trigger module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(GetCmdAddTransactionTrigger(), GetCmdAddBlockHeightTrigger(), GetCmdAddBlockTimeTrigger(), GetCmdDestroyTrigger())
```

Comment:
These are easier to read and maintain when put on different lines.

```suggestion
	txCmd.AddCommand(
		GetCmdAddTransactionTrigger(),
		GetCmdAddBlockHeightTrigger(),
		GetCmdAddBlockTimeTrigger(),
		GetCmdDestroyTrigger(),
	)
```

Reply:
Agreed